docs/source/scientific_papers.rst

```diff
@@ -0,0 +1,42 @@
+.. _scientific-papers:
+
+=================
+Scientific Papers
+=================
+
+The **signac** framework has been cited over 50 times, and has been used in a range of scientific fields with various types of computational workflows.
+Here, we showcase a few examples of scientific papers that have used **signac**.
+We encourage users to add their own publications to this list via a pull request.
+To find additional papers using **signac**, please refer to citations of the articles mentioned in :ref:`acknowledge`.
+
+- *Scalable Screening of Soft Matter: A Case Study of Mixtures of Ionic Liquids and Organic Solvents* :cite:`Thompson2019`
```

Comment:
This is a nice selection, but I think the formatting makes it actually very hard to parse. Maybe we can use something like [panels](https://sphinx-panels.readthedocs.io/) or some other visual divider to make the individual papers more distinguishable. The UMAP page has graphics for each paper, however I understand that it might be difficult to obtain permissions to represent abstract graphics for all listed papers.

I would also suggest to use a slightly more elaborate reference style with at least the journal and year or so. Something in between the representation of UMAP and HOOMD-blue. My assumption is that anyone who makes their way to that page won't be intimidated by the standard scientific reference format.

Reply:
@csadorf I changed the style pretty significantly. See if this looks better to you.

Reply:
Yes, that's much easier to parse. Now, the next question is, what is the order based on? Would anti-chronological make sense?

Reply:
Sure, I can do newest to oldest if that’s better.

Reply:
Yes, I think that would be best, but the previous order was not chronological in any way, which is why I was wondering what criterion you used.

Reply:
I updated the papers to be reverse-chronological. I didn't really have an order in mind before, the order was just copied from the listings in the 2021 SciPy Proceedings paper.